helm/botkube/values.yaml

```diff
@@ -135,260 +135,278 @@ sources:
-#              exclude: []
-#            # Overrides 'source'.kubernetes.event.types
-#            types:
-#              - create
-
-        - type: v1/services
-        - type: networking.k8s.io/v1/ingresses
-        - type: v1/nodes
-        - type: v1/namespaces
-        - type: v1/persistentvolumes
-        - type: v1/persistentvolumeclaims
-        - type: v1/configmaps
-        - type: rbac.authorization.k8s.io/v1/roles
-        - type: rbac.authorization.k8s.io/v1/rolebindings
-        - type: rbac.authorization.k8s.io/v1/clusterrolebindings
-        - type: rbac.authorization.k8s.io/v1/clusterroles
-        - type: apps/v1/daemonsets
-          event: # Overrides 'source'.kubernetes.event
-            types:
-              - create
-              - update
-              - delete
-              - error
-          updateSetting:
-            includeDiff: true
-            fields:
-              - spec.template.spec.containers[*].image
-              - status.numberReady
-        - type: batch/v1/jobs
-          event: # Overrides 'source'.kubernetes.event
-            types:
-              - create
-              - update
-              - delete
-              - error
-          updateSetting:
-            includeDiff: true
-            fields:
-              - spec.template.spec.containers[*].image
-              - status.conditions[*].type
-        - type: apps/v1/deployments
-          event: # Overrides 'source'.kubernetes.event
-            types:
-              - create
-              - update
-              - delete
-              - error
-          updateSetting:
-            includeDiff: true
-            fields:
-              - spec.template.spec.containers[*].image
-              - status.availableReplicas
-        - type: apps/v1/statefulsets
-          event: # Overrides 'source'.kubernetes.event
-            types:
-              - create
-              - update
-              - delete
-              - error
-          updateSetting:
-            includeDiff: true
-            fields:
-              - spec.template.spec.containers[*].image
-              - status.readyReplicas
-       ## Custom resource example
-       # - type: velero.io/v1/backups
-       #   namespaces:
-       #     include:
-       #       - ".*"
-       #     exclude:
-       #       -
-       #   event:
-       #     types:
-       #       - create
-       #       - update
-       #       - delete
-       #       - error
-       #   updateSetting:
-       #     includeDiff: true
-       #     fields:
-       #       - status.phase
+          # exclude: []
+
+        # -- Describes event constraints for Kubernetes resources.
+        # These constraints are applied for every resource specified in the `resources` list, unless they are overridden by the resource's own `events` object.
+        event:
+          # -- Lists all event types to be watched.
+          types:
+            - create
+            - delete
+            - error
```

Comment:
NOTE: after this merging this PR we won't merge configuration anymore. As a result, enabling both `k8s-err-events` and 'k8s-all-events' results in duplicated `error` messages. Probably will be good to mention that in the release notes/docs.

Reply:
Ok, I will be stating that in both release docs and main doc